public/css/showoff.css

```diff
@@ -927,6 +927,25 @@ form .element {
 ***  end callouts   ***
 **********************/
 
+/* Render hidden headlines so that when we print with wkhtmltopdf, we can use section
+   titles for page headers.  it would make sense to put this in the print section, but
+   then you get a weird double headline when previewing a print in the browser. */
+
+/* make this invisible as possible, while still rendering so that wkhtmltopdf doesn't ignore it */
+h1.section_title {
+  color: transparent !important;
+  opacity: 0 !important;
+  font-size: 0 !important;
+  margin: 0 !important;
+  padding: 0 !important;
+}
+
+/* because the above section headers are h1 elements rendered only in print */
```

Comment:
@marrero984 what's the real difference in intent between this and https://github.com/puppetlabs/showoff/blob/master/public/css/showoff.css#L14-L16 ?